src/Components/SmartComponents/SystemDetailsPage/SystemDetailsPage.js

```diff
@@ -41,46 +29,12 @@ class InventoryDetail extends React.Component {
         super(props);
 
         this.state = {
-            inventoryId: this.props.match.params.inventoryId,
-            removeListener: () => {}
+            inventoryId: this.props.match.params.inventoryId
         };
     }
 
     componentDidMount() {
         this.props.fetchSystemDetails(this.state.inventoryId);
-        this.fetchInventory();
```

Comment:
I was waiting for this moment since I joined the team. :grinning: :grinning:  :grinning: